app/jobs/application_job.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationJob < ActiveJob::Base
```

Comment:
It is the right time, but probably scope creep. Let's merge this PR and then open a new flow for standardising the ActiveJob stuff. Can you please add the card to Trello?